packages/generator/test/test-util/create-generator-options.ts

```diff
@@ -1,5 +1,8 @@
 import { GeneratorOptions } from '../../src/generator-options';
 
+/**
+ * @internal
+ */
```

Comment:
[req] Instead of marking everything in the tests internal it should be excluded from the check. Can we configure this?